src/server/middleware/error.ts

```diff
@@ -53,7 +53,17 @@ const isZodError = (err: unknown): boolean => {
 
 export const withErrorHandler = async (server: FastifyInstance) => {
   server.setErrorHandler(
-    (error: Error | CustomError | ZodError, request, reply) => {
+    (error: string | Error | CustomError | ZodError, request, reply) => {
+      if (typeof error === "string") {
```

Comment:
Currently a thrown string is not logged.
While we shouldn't throw raw strings, this update at least prints the string value.